src/benchmark.zig

```diff
@@ -234,48 +245,65 @@ const TimedQueue = struct {
         }
     }
 
-    pub fn lap(user_data: u128, operation: Operation, results: []const u8) void {
+    pub fn lap(user_data: u128, operation: Operation, results: ClientError![]const u8) void {
         const now = std.time.milliTimestamp();
-        assert(results.len == 0);
-        if (log_level == .debug) {
-            const response = std.mem.bytesAsSlice(CreateAccountsResult, results);
-            log.debug("{o}", .{response});
-        }
+        if (results) |value| {
+            if (log_level == .debug) {
```

Comment:
I think we don't need to check the `log_level` to skip the `log.debug()`, since the compiler will optimize the `debug()` call away (and I think also the `const response` dependency just before) if the log level is lower. The essentials for this are all comptime known.

Reply:
@jorangreef ok cool, I didn't know the compiler would do that :+1: